egs2/nit_song070/svs1/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+# spectrogram-related arguments
+fs=24000
+fmax=7600
+n_fft=2048
+n_shift=300
+win_length=1200
+
+score_feats_extract=frame_score_feats   # frame_score_feats | syllable_score_feats
+
+opts=
+if [ "${fs}" -eq 48000 ]; then
+    # To suppress recreation, specify wav format
+    opts="--audio_format wav "
+else
+    opts="--audio_format wav "
+fi
```

Comment:
We can directly use `opts="--audio_format wav "` here.

Reply:
Done.